packages/blade-old/src/atoms/TextInput/TextInput.web.js

```diff
@@ -534,15 +535,20 @@ const TextInput = ({
                                 </AccessoryText>
                               ) : null}
                               {hasRightIcon ? (
-                                <AccessoryIcon
-                                  variant={variant}
-                                  name={iconRight}
-                                  disabled={disabled}
-                                  hasError={hasError}
-                                  //@ts-expect-error
-                                  isFocused={isFocused}
-                                  position="right"
-                                />
+                                <View
```

Comment:
why do you need to wrap it in a view?

Reply:
We want `AccessoryIcon` to be clickable hence wrapping in a `View`. Do we have a better approach here? @kamleshchandnani

Reply:
Not a RN expert but can't we have clickable without a view?

Reply:
We can, but then we'd need to make changes to `AccessoryIcon` component. Simplest way here would be to wrap our `AccessoryIcon` with `TouchableOpacity` for RN and div for web.

Reply:
It's better to have a TouchableOpacity around it instead of making the icon clickable since TouchableOpacity will give you a onPress feedback by reducing the opacity when pressed. Making the SVG clickable will not give you a touch feedback

Reply:
This change should be fine for `TextInput.web.js` IMO, changes for `TextInput.native.js` is already done right @parichay28